modules/vsphere-problem-detector-running.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+[id="running-vsphere-problem-detector_{context}"]
+= Running the vSphere problem detector
+
+The vSphere Problem Detector Operator runs automatically with a default interval of 8 hours.
+To run the checks immediately, perform the following steps.
```

Comment:
You could replace "perform the following steps" with a high level description of what is required to run it manually. The reader might be uncertain as to why scaling the CSO down and back up again is required, especially if they don't realise that the vSphere Problem Detector Operator is storage specific at this stage.

In other words, it would be useful to the reader to explicitly state that they need to scale the CSO down to 0 and back to 1 and then outline why that triggers the checks.